src/components/Radios.tsx

```diff
@@ -0,0 +1,282 @@
+    describedBy += ` ${errorId}`;
+    errorMessageComponent = <ErrorMessage id={errorId} {...errorMessage} />;
+    if (formGroup) {
+      formGroup.error = true;
+    } else {
+      formGroup = { error: true };
+    }
+  }
+
+  const extraAttributes = Attributes.ariaAndData(props);
+  let classNames = "govuk-radios lbh-radios";
+  classNames += className ? ` ${className}` : "";
+  if (isConditional) {
+    classNames += " govuk-radios--conditional";
+  }
+
+  const innerContent = (
+    <>
+      {radiosHint && hintComponent}
+      {errorMessage && errorMessageComponent}
+      <div
+        className={classNames}
+        data-module={isConditional ? "govuk-radios" : undefined}
+        {...extraAttributes}
+      >
+        {items.map(
+          (item, index): JSX.Element => {
+            let id;
+            if (item.id) {
+              id = item.id;
+            } else {
+              if (index === 0) {
+                id = idPrefix;
+              } else {
+                id = `${idPrefix}-${index + 1}`;
+              }
+            }
+            const radioExtraAttributes = Attributes.ariaAndData(item);
+            if (item.divider) {
+              return (
+                <div className="govuk-radios__divider" key={index}>
+                  {item.divider}
+                </div>
+              );
+            } else {
+              const hasHint = item.hint && item.hint.children ? true : false;
+              const itemHintId = `${id}-item-hint`;
+              let labelClasses = "govuk-radios__label";
+              labelClasses +=
+                item.label && item.label.className ? item.label.className : "";
+              const conditionalClasses =
+                "govuk-radios__conditional" + item.checked
+                  ? ""
+                  : " govuk-radios__conditional--hidden";
+
+              return (
+                <React.Fragment key={index}>
+                  <div className="govuk-radios__item">
+                    <input
+                      className="govuk-radios__input"
+                      id={id}
+                      name={name}
+                      type="radio"
+                      value={item.value}
+                      checked={item.checked ? true : undefined}
+                      disabled={item.disabled ? true : undefined}
+                      data-aria-controls={
+                        item.conditional && item.conditional.html
+                          ? `conditional-${id}`
+                          : undefined
+                      }
+                      aria-describedBy={hasHint ? itemHintId : undefined}
+                      {...radioExtraAttributes}
+                      onChange={
+                        onChange !== undefined
+                          ? (): void => {
+                              onChange(item.value as string);
+                            }
+                          : undefined
+                      }
```

Comment:
This is fine, but it would probably be more readable and idiomatic as:

```
(): void => {
  if (onChange) {
    onChange(item.value);
  }
}
```

Reply:
If I do this then the output in the test has `onChange={[Function]}`

Reply:
That's fair, and not something I'd considered actually.